src/webextension/widgets/scrolling-list.css

```diff
@@ -16,10 +16,15 @@
 
 .scrolling-list > li[data-selected] {
   color: #0c0c0d;
-  background-color: #b1b1b3;
+  background-color: #f9f9fa;
+  border-right: none;
```

Comment:
>This should probably go in item-summary.js.

I had a hard time getting all of this sidebar / scroll list properly targeted. Are you saying the border removal itself should be removed from here specifically because its reused in the doorhanger?

Reply:
Generally, yeah. The generic `<ScrollingList/>` should have a minimum of CSS (even the CSS for selected rows might be too much, but I wanted it to work ok out of the box).